chore(deps): upgrade dependencies

cdk8s-automation opened the pull request:
Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/cdk8s-team/cdk8s-plus/actions/runs/1299366445

------

*Automatically created by projen via the "upgrade" workflow*

Chriscbr commented:
Looks like the build is failing because Amazon.JSII.Runtime 1.35.0 has not been released yet on NuGet gallery. I'll wait a few days to see if this is temporary.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/cdk8s-team/cdk8s-plus/actions/runs/1299366445

------

*Automatically created by projen via the "upgrade" workflow*

Commit:
manual fixes
